src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -858,7 +858,8 @@ public function testGetClientIpsForwardedProvider()
         //              $expected                                  $remoteAddr  $httpForwarded                                       $trustedProxies
         return array(
             array(array('127.0.0.1'),                              '127.0.0.1', 'for="_gazonk"',                                      null),
-            array(array('_gazonk'),                                '127.0.0.1', 'for="_gazonk"',                                      array('127.0.0.1')),
+            array(array('127.0.0.1'),                              '127.0.0.1', 'for="_gazonk"',                                      array('127.0.0.1')),
```

Comment:
This test case is modified, but returning something different that an IP address here looks broken.